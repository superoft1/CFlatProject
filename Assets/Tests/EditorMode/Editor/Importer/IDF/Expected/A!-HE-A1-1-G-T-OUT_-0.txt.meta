fileFormatVersion: 2
guid: ad52e7f7162c4c24683486029bddc01b
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
