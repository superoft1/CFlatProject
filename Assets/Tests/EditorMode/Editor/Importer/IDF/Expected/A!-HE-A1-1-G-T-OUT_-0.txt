WeldNeckFlange, , (0.0, 0.0, -1.0), (-372.2, 14.5, 4.5), False
(-372.2, 14.5, 4.4)
(-372.2, 14.5, 4.6)
FreeVertex: 1
Pipe, , (0.0, 0.0, 1.0), (-372.2, 14.5, 4.7), False
(-372.2, 14.5, 4.6)
(-372.2, 14.5, 4.7)
FreeVertex: 0
Elbow90, , (-1.0, 0.0, 0.0), (-372.2, 14.5, 5.6), False
(-372.2, 14.5, 4.7)
(-371.3, 14.5, 5.6)
FreeVertex: 0
Pipe, , (1.0, 0.0, 0.0), (-371.2, 14.5, 5.6), False
(-371.3, 14.5, 5.6)
(-371.1, 14.5, 5.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-371.0, 14.5, 5.6), False
(-371.0, 14.5, 5.6)
(-371.1, 14.5, 5.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-370.9, 14.5, 5.6), False
(-371.0, 14.5, 5.6)
(-370.8, 14.5, 5.6)
FreeVertex: 1
FreeVertexTotal: 2
